drivers/overlay/peerdb.go

```diff
@@ -244,13 +237,12 @@ func (d *driver) peerDbUpdateSandbox(nid string) {
 
 		peerOps = append(peerOps, op)
 	}
-	pMap.Unlock()
 
 	for _, op := range peerOps {
 		op()
 	}
+	pMap.Unlock()
```

Comment:
@mavenugo On the locking, the scope of the lock is changed only in `peerDbUpdateSandbox` which was already serialized using the WaitGroup. So that looks ok. As I mentioned in the earlier comment its good to keep the old logic for serf mode till its deprecated.